src/nativeTrackerManager.js

```diff
@@ -10,6 +10,7 @@ const AD_DATA_ADID_ATTRIBUTE = 'pbAdId';
 export function newNativeTrackerManager(win) {
   let publisherDomain;
 
+
```

Comment:
extra new line?

Reply:
Removing this. I believe I originally added logic that I later removed.